docs/Database.md

```diff
@@ -0,0 +1,78 @@
+TODO: Move to code using http://usejsdoc.org/ format.
```

Comment:
Should mostly be in the code already but we can have this around for now just in case. Probably wouldn't hurt to drop it, though.

Reply:
want to roughly start this with `mv trustroots.wiki/*md trustroots/docs/` - and then clean that up afterwards.

first gotta grok pull requests tho https://trustroots.slack.com/archives/C0A3Q15SS/p1542227913019400

Reply:
makes sense :thumbsup: